pykeg/backend/backends.py

```diff
@@ -656,6 +656,21 @@ def connect_toggle(self, tap, new_toggle):
 
         return tap
 
+    @transaction.atomic
+    def connect_thermo(self, tap, new_thermo):
+        tap = self._get_tap(tap)
+        old_thermo = tap.temperature_sensor
+
+        if old_thermo != new_thermo:
+            if old_thermo:
+                tap.temperature_sensor = None
+                tap.save()
+            if new_thermo:
+                tap.temperature_sensor = new_thermo
+                tap.save()
```

Comment:
I'd think something like this would be cleaner:
```
if new_thermo:
  tap.temperature_sensor = new_thermo
else:
  tap.temperature_sensor = None
tap.save()
```
Unless the tap needs to be saved without a thermo before it can be saves with a thermo.